pwnlib/abi.py

```diff
@@ -71,28 +72,29 @@ def syscall():
             context.os = 'linux'
 
         return {
-        (32, 'i386', 'linux'):  linux_i386_syscall,
-        (64, 'amd64', 'linux'): linux_amd64_syscall,
-        (64, 'aarch64', 'linux'): linux_aarch64_syscall,
-        (32, 'arm', 'linux'):   linux_arm_syscall,
-        (32, 'thumb', 'linux'):   linux_arm_syscall,
-        (32, 'mips', 'linux'):   linux_mips_syscall,
-        (64, 'aarch64', 'linux'):   linux_aarch64_syscall,
-        (32, 'powerpc', 'linux'): linux_ppc_syscall,
-        (64, 'powerpc', 'linux'): linux_ppc64_syscall,
-        (32, 'riscv32', 'linux'): linux_riscv32_syscall,
-        (64, 'riscv64', 'linux'): linux_riscv64_syscall,
-        (32, 'i386', 'freebsd'):  freebsd_i386_syscall,
-        (64, 'amd64', 'freebsd'): freebsd_amd64_syscall,
-        (64, 'aarch64', 'freebsd'): freebsd_aarch64_syscall,
-        (32, 'arm', 'freebsd'):   freebsd_arm_syscall,
-        (32, 'thumb', 'freebsd'):   freebsd_arm_syscall,
-        (32, 'mips', 'freebsd'):   freebsd_mips_syscall,
-        (64, 'aarch64', 'freebsd'):   freebsd_aarch64_syscall,
-        (32, 'powerpc', 'freebsd'): freebsd_ppc_syscall,
-        (64, 'powerpc', 'freebsd'): freebsd_ppc64_syscall,
-        (64, 'amd64', 'darwin'): darwin_amd64_syscall,
-        (64, 'aarch64', 'darwin'): darwin_aarch64_syscall,
+        (32, "i386", "linux"): linux_i386_syscall,
+        (64, "amd64", "linux"): linux_amd64_syscall,
+        (64, "aarch64", "linux"): linux_aarch64_syscall,
+        (32, "arm", "linux"): linux_arm_syscall,
+        (32, "thumb", "linux"): linux_arm_syscall,
+        (32, "mips", "linux"): linux_mips_syscall,
+        (64, "aarch64", "linux"): linux_aarch64_syscall,
+        (32, "powerpc", "linux"): linux_ppc_syscall,
+        (64, "powerpc", "linux"): linux_ppc64_syscall,
+        (32, "riscv32", "linux"): linux_riscv32_syscall,
+        (64, "riscv64", "linux"): linux_riscv64_syscall,
+        (64, "loong64", "linux"): linux_loong64_syscall,
+        (32, "i386", "freebsd"): freebsd_i386_syscall,
+        (64, "amd64", "freebsd"): freebsd_amd64_syscall,
+        (64, "aarch64", "freebsd"): freebsd_aarch64_syscall,
+        (32, "arm", "freebsd"): freebsd_arm_syscall,
+        (32, "thumb", "freebsd"): freebsd_arm_syscall,
+        (32, "mips", "freebsd"): freebsd_mips_syscall,
+        (64, "aarch64", "freebsd"): freebsd_aarch64_syscall,
+        (32, "powerpc", "freebsd"): freebsd_ppc_syscall,
+        (64, "powerpc", "freebsd"): freebsd_ppc64_syscall,
+        (64, "amd64", "darwin"): darwin_amd64_syscall,
+        (64, "aarch64", "darwin"): darwin_aarch64_syscall,
```

Comment:
please avoid changing single quotes to double quotes without any reasons.
It makes the diff bigger, and hard to spot the actually changes.